.github/workflows/nodejs-publish.yaml

```diff
@@ -21,6 +33,12 @@ jobs:
         with:
           node-version: ${{ inputs.node_version }}
           registry-url: 'https://registry.npmjs.org'
+      - name: Download build artifact
+        if: inputs.download_artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.build_folder }}
+          path: ${{ inputs.build_folder_path }}
       - name: Publish to npm.js
         run: npm publish --access=public
```

Comment:
I think it's a good idea to pass this through as an optional `args` input and pass it through from the validation package workflow.
Then people could use this workflow for private libraries.